lib/services/serviceFabric/README.md

```diff
@@ -9,7 +9,7 @@ This project provides a Node.js package for accessing the Azure Service Fabric.
 ## How to Install
 
 ```bash
-npm install azure-servicefabric
+npm install azure-arm-servicefabric
```

Comment:
why? you are not an ARM service and you do not use Azure.NodeJS version of the codegen. So the package name azure-servicefabric is just perfect.